src/fiatconnect/selectors.ts

```diff
@@ -20,3 +21,8 @@ export const cachedQuoteParamsSelector = (state: RootState) => state.fiatConnect
 export const schemaCountryOverridesSelector = (state: RootState) =>
   state.fiatConnect.schemaCountryOverrides
 export const personaInProgressSelector = (state: RootState) => state.fiatConnect.personaInProgress
+export const getCachedFiatConnectTransfer =
```

Comment:
nit: Looks like all methods end with the name Selector.. maybe `getCachedFiatConnectTransferSelector` or `getCachedTransferSelector`?

Reply:
Ok, updated to `getCachedFiatConnectTransferSelector`